misc/changelog-generator/instructions.md

```diff
@@ -0,0 +1,30 @@
+
+### Creating Pull Requests for Inclusion in NEWS.md
+
+If a PR is significant enough to warrant a mention in the next release notes update,
+its name should begin with a prefix. 
+There are three options depending on the PR's
+purpose.
+
+  * `breaking-change: ` if the change will make the next tag non-backward-compatible
+  * `feature: ` if the change is a major addition that maintains backward-compatibility
+  * `fix: ` if the change is a bug fix, security patch, or other improvement
+
+If the PR does not begin with one of these prefixes, it WILL NOT be included in
+the release notes, so make sure to name important PRs accordingly. 
+
```

Comment:
@diondrapeck should we add a prefix for documentation?

Reply:
good point - i'll add a "documentation: " tag that won't affect the version number.